src/utils/trusts.ts

```diff
@@ -275,10 +292,17 @@ const saveHistoricalBoInTrust = (
 const getLegalEntityBosInTrust = (
   appData: ApplicationData,
   trustId?: string,
+  isReview?: boolean,
 ): TrustCorporate[] => {
   let legalEntities: TrustCorporate[] = [];
+  let trustList: Trust[] | undefined;
+  if (isReview) {
+    trustList = appData[UpdateKey]?.[ReviewTrustKey];
+  } else {
+    trustList = appData[TrustKey];
+  }
   if (trustId) {
-    legalEntities = appData[TrustKey]?.find(trust =>
+    legalEntities = trustList?.find(trust =>
       trust?.trust_id === trustId)?.CORPORATES as TrustCorporate[];
```

Comment:
"trust.trust_id == trustId" should work. Should we be calling find on "trustList?" as we could check for that being undefined on 280 say?

Reply:
Do you mean we should check if trustList is not undefined before we call find on it? Like this:
`  if (trustId && trustList) {
    legalEntities = trustList.find(trust =>
      trust?.trust_id == trustId)?.CORPORATES as TrustCorporate[];`

Reply:
yes, that's it. But no "as TrustCorporate[]". Saver to check for undefined which you do on line 283 anyway.

Reply:
again this is just a variable refactor by Christy, actual logic is existing code